Makefile

```diff
@@ -1,6 +1,10 @@
 manage = python manage.py
 install = pip install
 
+docker:
+	sed -e 's/-r common.txt//' requirements/dev.txt \
+	  | cat requirements/common.txt - > requirements.txt
```

Comment:
It isn't exactly Docker that copies this file, but the Django base image. But yes, it gets copied inside so it has to be available.

One might use a different base image to drop this requirement, though.

I will try to find a better approach then.